tx/textual/valuerenderer/any_test.go

```diff
@@ -0,0 +1,61 @@
+package valuerenderer_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"cosmossdk.io/tx/textual/valuerenderer"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/require"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/testing/protocmp"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type anyJsonTest struct {
+	Proto   map[string]interface{}
+	Screens []valuerenderer.Screen
+}
+
+func TestAny(t *testing.T) {
+	raw, err := os.ReadFile("../internal/testdata/any.json")
+	require.NoError(t, err)
+
+	var testcases []anyJsonTest
+	err = json.Unmarshal(raw, &testcases)
+	require.NoError(t, err)
+
+	tr := valuerenderer.NewTextual(EmptyCoinMetadataQuerier)
+	for i, tc := range testcases {
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
+			// Must marshal the proto object back into JSON,
+			// then unmarshal with protojson into an Any.
+			bz, err := json.Marshal(tc.Proto)
+			require.NoError(t, err)
```

Comment:
I propose to change `tc.Proto` be `json.RawMessage`, and we don't need json.Marshal anymore

```suggestion
```

Reply:
Done. Good catch, thanks!